react-native-libraries.json

```
@@ -10340,5 +10340,18 @@
     "android": true,
     "web": true,
     "newArchitecture": true
+  },
+  {
+    "nameOverride": "react-native-image-preview-reanimated",
+    "githubUrl": "https://github.com/sbaiahmed1/react-native-image-preview",
+    "examples": [
+      "https://github.com/sbaiahmed1/react-native-image-preview/tree/main/example"
+    ],
+    "npmPkg":"@sbaiahmed1/react-native-image-preview-reanimated",
```

Hello @sbaiahmed1, thank you for adding new library to the directory! 👍 

It looks like the package has been published under [`react-native-image-preview-reanimated`](https://www.npmjs.com/package/react-native-image-preview-reanimated) name, so I have altered entry to reflect that.

```suggestion
    "githubUrl": "https://github.com/sbaiahmed1/react-native-image-preview",
    "npmPkg":"react-native-image-preview-reanimated",
    "examples": [
      "https://github.com/sbaiahmed1/react-native-image-preview/tree/main/example"
    ],
```

With this change in, the PR should be ready to ship.